recipes/cminpack/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.3.6" %}
+
+package:
+  name: cminpack
+  version: {{ version }}
+
+source:
+  url: https://github.com/devernay/{{ name }}/archive/v{{ version }}.zip
+  sha256: 99865735c21b8c035acfcf9390e7e69efacc6389863044e516b0ae57c8c539f5
+
+build:
+  skip: true  # [win and vc<14]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}  # Needed because CMake project is not declared as C
+    - cmake
+    - ninja
+
+test:
+  commands:
+    - if not exist %LIBRARY_LIB%\\cminpack.lib exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\\cminpack.dll exit 1  # [win]
+    - if not exist %LIBRARY_INC%\\cminpack-1\\cminpack.h exit 1  # [win]
+    - if not exist %LIBRARY_INC%\\cminpack-1\\minpack.h exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\\share\\cmake\\CMinpack\\CMinpackConfig.cmake  # [win]
+
+    - test -f $PREFIX/lib/libcminpack.so  # [unix]
+    - test -f $PREFIX/lib/libcminpack.so.{{ version }}  # [unix]
+    - test -f $PREFIX/include/cminpack-1/cminpack.h  # [unix]
+    - test -f $PREFIX/include/cminpack-1/minpack.h  # [unix]
+    - test -f $PREFIX/share/cmake/CMinpack/CMinpackConfig.cmake  # [unix]
+
+about:
+  home: http://devernay.free.fr/hacks/cminpack/index.html
+  license: MINPACK  # Custom BSD-like
+  license_family: BSD
+  summary: 'CMinpack includes software for solving nonlinear equations and nonlinear least squares problems'
+  license_file: CopyrightMINPACK.txt
+  dev_url: https://github.com/devernay/cminpack
+
+extra:
+  recipe-maintainers:
+    - tadeu
+    - devernay
```

Comment:
@devernay, is it ok to add you as a maintainer here?

Reply:
I maintain the source, but I can't maintain the conda package

Reply:
It's alright, thanks!